doc/OneStepSetup.md

```diff
@@ -14,8 +14,8 @@ This is a streamlined process for setting up the Pi. You'll flash a preconfigure
 
     In Raspberry Pi Imager, you need to click 'Operating System' and then scroll _all the way down_ and select the 'Use custom' option.
 
-1.  Mount the card again, and in the `boot` directory create a `teslausb_setup_variables.conf` file to export the same environment variables normally needed for manual setup (including archive info, Wifi, and push notifications (if desired).
-    A sample conf file is located in the `boot` folder on the SD card. The latest sample is also available [from GitHub](https://github.com/marcone/teslausb/blob/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample).
+1.  Mount the 'boot'-labeled partition on the card again, and create a `teslausb_setup_variables.conf` file to export the same environment variables normally needed for manual setup (including archive info, Wifi, and push notifications (if desired).
+    A sample conf file is located in the `boot` partition on the SD card. The latest sample is also available [from GitHub](https://github.com/marcone/teslausb/blob/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample). Do not confuse the 'boot' parition with the 'boot' directory on the 'rootfs' partition!
```

Comment:
That added sentence doesn't seem very useful. Most people won't see the root partition. Referring to something that doesn't even exist as far as most people are concerned will just lead to confusion.